actualbudget/umbrel-app.yml

```diff
@@ -0,0 +1,34 @@
+manifestVersion: 1
+id: actualbudget
+category: Finance
+name: ActualBudget
+version: "23.5.0"
+tagline: Envelope budgeting made easy
+description: >-
+  Actual is a super fast privacy-focused app for managing your finances. 
+  You own your data, and we will sync it across all devices with optional end-to-end encryption.
+
+  It provides a budgeting system based on "envelope budgeting," a proven method for managing your spending. 
+  You can also track all of your finances, including investment accounts, in one place.
+
+  Download the mobile application on the App Store and Play Store, then link it to you umbrel instance!
+
+  ⚠️ Important ⚠️
+  When installing the application for the first time, a warning will appear. Click on "Advanced Options", tick the box and click "Open Actual".
+  This is because ActualBudget is served via HTTP. We recommend no opening more than one tab to avoid data loss risk.
```

Comment:
Here's the warning for our reference:

<img width="509" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/862141de-1d9d-4986-89a4-eadf10884ff6">

<img width="616" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/8ed6c3a7-fc15-4fd1-ba33-6ec42b942043">